cli/types/cypress.d.ts

```diff
@@ -1815,6 +1815,17 @@ declare namespace Cypress {
      *    cy.reload(true)
      */
     reload(forceReload: boolean): Chainable<AUTWindow>
+    /**
+     * Reload the page without cache and with log and timeout options
+     *
+     * @see https://on.cypress.io/reload
+     * @param {Boolean} forceReload Whether to reload the current page without using the cache. true forces the reload without cache.
```

Comment:
Should add an additional `@param` for the options parameter and update the example section

Reply:
Hopefully I got it right 🤞

Reply:
![Screenshot 2023-02-14_11-35-50](https://user-images.githubusercontent.com/78863563/218713945-8d035ac6-f807-4dcf-8d1d-6a9a859b5985.jpg)
![Screenshot 2023-02-14_11-36-09](https://user-images.githubusercontent.com/78863563/218713949-b9fb3064-193c-418e-812d-75f5e3655a7c.jpg)
![Screenshot 2023-02-14_11-36-18](https://user-images.githubusercontent.com/78863563/218713953-658703f6-f57c-4e8a-8737-255b4b981eb0.jpg)
![Screenshot 2023-02-14_11-36-28](https://user-images.githubusercontent.com/78863563/218713957-9b041bb8-000d-4b12-a579-17e598e1b9bb.jpg)